src/docs/Updates/Roadmap/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Roadmap'
+order: 2
+description: 'Discover what we are working on right now'
+---
+
+## Q1 2020 (from 01/01/2020 to 01/04/2020)
+
+### Create a bêta version of the Mozaic-vue library
+
+- Implement CI / CD and code quality tools **(WIP)**
+- Create contributing and coding guidelines **(WIP)**
+- Create and migrate at least all the forms components existing in Mozaic into Mozaic-vue **(WIP)**
+- Link Mozaic-vue in the Mozaic website
+- Support contribution from other teams
+
+<br />
+
+---
+
+### Make contributing easy
+
+- Organising contributions **(WIP)**
+- Improve documentation about contributting **(WIP)**
+- Create a contributing funnel **(WIP)**
+- Reduce PR and merge time as much as possible
+- Set up a voting system for missing patterns
+- Following and supporting contributors to improve Mozaic
+
+<br />
+
+---
+
+### Improve Mozaic user experience
+
```

Comment:
- Homepage redesign
- Website menu redesign
- Pattern page and table of content redesign
- Add documentation search engine

Reply:
ok